data/json/overmap/specials.json

```diff
@@ -2379,11 +2377,13 @@
       { "point": [ 0, 1, 0 ], "overmap": "s_reststop_1_north" },
       { "point": [ 1, 1, 0 ], "overmap": "s_reststop_2_north" }
     ],
-    "connections": [ { "point": [ 0, -1, 0 ], "terrain": "road" } ],
+    "connections": [ { "point": [ 0, -1, 0 ], "terrain": "road", "existing": true },
+                     { "point": [ 1, -1, 0 ], "terrain": "road", "existing": true }
+                   ],
     "locations": [ "wilderness" ],
     "city_distance": [ 10, 150 ],
     "city_sizes": [ 1, 12 ],
-    "occurrences": [ 0, 1 ],
+    "occurrences": [ 1, 2 ],
```

Comment:
I'm not sure I'd make a rest stop a required OM feature (remember, there are only so many slots on roads for OM specials). I do agree with increasing the limit to 2 though.